### ISSUE TITLE
allow closing svg tag like <cirlcle ... />

### ISSUE THREAD

**Issue by SimonHeimberg** [thibaudcolas/curlylint#4]
### Is your proposal related to a problem?

To directly closing some svg tags (at least `<circle .../>`) is not allowed:
```twig
<svg width="20" height="20">
    <circle cx="10" cy="10" r="10" stroke-width="1" stroke="black" fill="{{ entry.getStatusColor() }}" opacity="0.5" />
</svg>
```

this error is reported:
```console
tmp/xxx.html.twig:2:117 Parse error: expected one of '>', 'attribute', '{#', '{%', '{{' at 1:117
```

### Describe the solution you’d like

In case the standard forbids this, I am fine. But for me https://developer.mozilla.org/en-US/docs/Web/SVG/Element/circle looks like it is allowed.

So I suppose to not report this as failure.

### Describe alternatives you’ve considered

--

### Additional context

--

**Comment by SimonHeimberg** [comment]
Wait, this is fixed in [v0.10.0](https://github.com/thibaudcolas/curlylint/releases/tag/v0.10.0). Unfortunately I have to use v0.6 because of an old python version... 😞

**Comment by thibaudcolas** [comment]
Hey @SimonHeimberg, yep, that makes sense. Can I ask why you’re stuck with Python 3.5? A lot of projects have started dropping support for it (black, Django), because its end-of-life date is only a few months away (https://devguide.python.org/#status-of-python-branches).

I think this one SVG fix would be quite easy to support, but there’s lots more improvements I’ve made since releasing v0.6 – adding ways to ignore files, linting rules, etc. Essentially the only thing you’d be getting with v0.6 is the parser throwing an error if there is invalid markup in your files – and then again the parser is far from finished.
